src/lib/utils/notifications/push.ts

```diff
@@ -1,5 +1,5 @@
 import { error } from "@sveltejs/kit";
-import Expo, { type ExpoPushMessage } from "expo-server-sdk";
+import { Expo, type ExpoPushMessage } from "expo-server-sdk";
 import { dev } from "$app/environment";
 
 async function sendPushNotifications(
```

Comment:
I know this is not from this PR, but I REALLY think we should enable push notification security and use our expo private key here. As it is right now, given someones expo push notification ANYONE could send a push notification to anyone of our users.

Reply:
Sounds like a good idea. Unfortunately don't know how to do this, would be great if you could do a commit with the required changes.